test/test-util.js

```diff
@@ -6,203 +6,160 @@
-  assertUrl("http://www.example.com/foo/bar");
-  assertUrl("http://www.example.com/foo/bar/");
-  assertUrl("http://user:pass@www.example.com:80/foo/bar/");
-
-  assertUrl("//");
-  assertUrl("//www.example.com");
-  assertUrl("file:///www.example.com");
-
-  assert.equal(libUtil.urlParse(""), null);
-  assert.equal(libUtil.urlParse("."), null);
-  assert.equal(libUtil.urlParse(".."), null);
-  assert.equal(libUtil.urlParse("a"), null);
-  assert.equal(libUtil.urlParse("a/b"), null);
-  assert.equal(libUtil.urlParse("a//b"), null);
-  assert.equal(libUtil.urlParse("/a"), null);
-  assert.equal(libUtil.urlParse("data:foo,bar"), null);
-
-  let parsed = libUtil.urlParse("http://x-y.com/bar");
-  assert.equal(parsed.scheme, "http");
-  assert.equal(parsed.host, "x-y.com");
-  assert.equal(parsed.path, "/bar");
-
-  const webpackURL = "webpack:///webpack/bootstrap 67e184f9679733298d44";
-  parsed = libUtil.urlParse(webpackURL);
-  assert.equal(parsed.scheme, "webpack");
-  assert.equal(parsed.host, "");
-  assert.equal(parsed.path, "/webpack/bootstrap 67e184f9679733298d44");
-  assert.equal(webpackURL, libUtil.urlGenerate(parsed));
-};
-
 exports["test normalize()"] = function(assert) {
   assert.equal(libUtil.normalize("/.."), "/");
   assert.equal(libUtil.normalize("/../"), "/");
   assert.equal(libUtil.normalize("/../../../.."), "/");
   assert.equal(libUtil.normalize("/../../../../a/b/c"), "/a/b/c");
   assert.equal(libUtil.normalize("/a/b/c/../../../d/../../e"), "/e");
 
-  assert.equal(libUtil.normalize(".."), "..");
+  assert.equal(libUtil.normalize(".."), "../");
   assert.equal(libUtil.normalize("../"), "../");
+
   assert.equal(libUtil.normalize("../../a/"), "../../a/");
-  assert.equal(libUtil.normalize("a/.."), ".");
-  assert.equal(libUtil.normalize("a/../../.."), "../..");
+  assert.equal(libUtil.normalize("a/.."), "");
+  assert.equal(libUtil.normalize("a/../../.."), "../../");
 
   assert.equal(libUtil.normalize("/."), "/");
   assert.equal(libUtil.normalize("/./"), "/");
   assert.equal(libUtil.normalize("/./././."), "/");
   assert.equal(libUtil.normalize("/././././a/b/c"), "/a/b/c");
   assert.equal(libUtil.normalize("/a/b/c/./././d/././e"), "/a/b/c/d/e");
 
-  assert.equal(libUtil.normalize(""), ".");
-  assert.equal(libUtil.normalize("."), ".");
-  assert.equal(libUtil.normalize("./"), ".");
+  assert.equal(libUtil.normalize(""), "");
+  assert.equal(libUtil.normalize("."), "");
+  assert.equal(libUtil.normalize("./"), "");
   assert.equal(libUtil.normalize("././a"), "a");
   assert.equal(libUtil.normalize("a/./"), "a/");
-  assert.equal(libUtil.normalize("a/././."), "a");
+  assert.equal(libUtil.normalize("a/././."), "a/");
 
-  assert.equal(libUtil.normalize("/a/b//c////d/////"), "/a/b/c/d/");
-  assert.equal(libUtil.normalize("///a/b//c////d/////"), "///a/b/c/d/");
-  assert.equal(libUtil.normalize("a/b//c////d"), "a/b/c/d");
+  assert.equal(libUtil.normalize("/a/b//c////d/////"), "/a/b//c////d/////");
```

Comment:
Not sure it matters but this doesn't seem like a "normal" form.  What's the deal here?